src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,28 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movie")
 public class Movie implements Cloneable {
```

Comment:
There is no need to implement `Cloneable` interface and override `clone` method. It's not a common practice in Java and especially in the context of Hibernate entities. It can lead to unexpected behavior. You should remove `Cloneable` and `clone` method.